SHA256.md

```diff
@@ -32,4 +32,4 @@ d7b0af158a166d1c89fc91845fb7baa1e0d46da71e116e9b6aaf44ef5cb1e83b  salesforcedx-v
 5. In Visual Studio Code, from the Extensions view, select ... > Install from
 VSIX.
 
-6. Install the verified VSIX file.
+6. Install the verified VSIX file. 
```

Comment:
I have no idea what is going on but I can't get this file out of the PR. I've tried 3 times now. I give up